op-e2e/actions/proofs/pectra_blob_schedule_test.go

```diff
@@ -27,7 +27,7 @@ func TestPectraBlobSchedule(gt *testing.T) {
 	matrix.AddDefaultTestCases(
 		// aligned with an L1 timestamp
 		pectraBlobScheduleTestCfg{ptr(uint64(24)), true},
-		helpers.NewForkMatrix(helpers.Holocene),
+		helpers.NewForkMatrix(helpers.Holocene, helpers.LatestFork),
```

Comment:
I'd prefer to be explicit here and run it on Isthmus, so it runs for Holocene+Isthmus. We won't need to run this test for forks later than Isthmus (when the `LatestFork` pointer changes in the future), so let's just pin both forks.
```suggestion
		helpers.NewForkMatrix(helpers.Holocene, helpers.Isthmus),
```